src/components/OmniSearch/__tests__/__snapshots__/OmniSearch.test.tsx.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@Kushagra8888 - any clue why this is happening instead of the proper snapshot?

Reply:
Not sure... try using a plain div in the render.. maybe the dialog could be creating problems as it may be rendered in a portal?